ecommerce-ui/src/Components/Map/Map.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useCallback, useMemo } from "react";
+import PropTypes from "prop-types";
+import "./Map.css";
+
+import { GoogleMap, useJsApiLoader, Marker } from "@react-google-maps/api";
+
+const containerStyle = {
+  height: "240px",
+};
+
+export default function Map({ lat, lng }) {
+  const center = useMemo(() => {
+    return {
+      lat: lat,
+      lng: lng,
+    };
+  }, [lat, lng]);
```

Comment:
This works fine. It'll also work if you pass `{lat, lng}` directly where you're passing `center`, but that's a bit repetitive. Otherwise, `useCallback` will throw a lint error wanting you to use `useMemo` for `center`.